src/docfx/build/dependency/LinkResolver.cs

```diff
@@ -173,7 +173,7 @@ internal class LinkResolver
 
                     // resolve file
                     var lookupFallbackCommits = inclusion || _documentProvider.GetContentType(path) == ContentType.Resource;
-                    var file = TryResolveRelativePath(referencingFile.FilePath, path, lookupFallbackCommits);
+                    var file = TryResolveRelativePath(referencingFile.Docset.DocsetPath, referencingFile.FilePath, path, lookupFallbackCommits);
```

Comment:
Add a `string _docsetPath` member field and initialize it from constructor?

Reply:
It's just a private method, do not feel like a private field is needed for it.